### DISCUSSION TITLE
Any partners to help set up?

### DISCUSSION THREAD

**Discussion by osseonews** [vendurehq/vendure#914] [Q&A]
Really interested in setting this up and trying out the NextJs frontend. Curious if anyone reading this board can help us set up Vendure on Digital Ocean with postgres database? We only front end developers right now and setting this up on DO has proven way beyond our capabilities. Obviously, we would pay for set up. Thanks!

**Comment by clingovdev** [comment]
I can help, I have a full setup running on a CI/CD pipeline from Azure Repos to an Azure App service running a postgres database as separated concerns.

cliff@infinityarc.net

**Reply by CliffCrerar** [reply]
Real GitHub profile.

**Reply by osseonews** [reply]
thanks. Will shoot you an email.
